README.md

```diff
@@ -77,8 +82,8 @@ yarn axe
 
 ## 👤 Author
 
-- Github: [@author](https://github.com/author)
-- Twitter: [@author](https://twitter.com/author)
+- Github: [@author](https://github.com//Gbemisola19)
+- Twitter: [@author](https://twitter.com/OladipoAzeezat)
 - Linkedin: [@author](https://www.linkedin.com/in/author/)
```

Comment:
Let's change all the `@author` to an actual handle or profile username instead. If you don't have a LinkedIn profile, you can remove it from the Readme.